airflow/operators/bash.py

```diff
@@ -63,6 +63,8 @@ class BashOperator(BaseOperator):
         If None (default), the command is run in a temporary directory.
         To use current DAG folder as the working directory,
         you might set template ``{{ dag_run.dag.folder }}``.
+    :param result_processor: Function to further process the result of the bash script
```

Comment:
I'd rather call it `output_processor` to be consistent with terminology (i.e., existing `output_encoding`)

Reply:
I'm a bit confused - do you agree with the renaming? (it is not implemented in practice)

Reply:
I thought I've pushed my changes but didn't, now it should be ok

Reply:
I completely agree on the renaming ;)